beacon-chain/core/balances/rewards_penalties.go

```diff
@@ -345,12 +345,9 @@ func Crosslinks(
 	endSlot := helpers.StartSlot(currentEpoch)
 
 	for i := startSlot; i < endSlot; i++ {
-		var registryChange bool
-		if state.ValidatorRegistryUpdateEpoch == i-1 &&
-			state.ValidatorRegistryUpdateEpoch != params.BeaconConfig().GenesisEpoch {
-			registryChange = true
-		}
-		crosslinkCommittees, err := helpers.CrosslinkCommitteesAtSlot(state, i, registryChange)
+		// RegistryChange is a no-op when requesting slot in current and previous epoch.
+		// Process crosslinks rewards will never request crosslink committees of next epoch.
+		crosslinkCommittees, err := helpers.CrosslinkCommitteesAtSlot(state, i, false) //registryChange = false
```

Comment:
Please use this syntax `helpers.CrosslinkCommitteesAtSlot(state, i, false /* registryChange */)`